helm_sdk/utils.py

```diff
@@ -152,3 +154,10 @@ def prepare_set_parameters(set_values):
             raise CloudifyHelmSDKError(
                 "set parameter name or value is missing")
     return set_list
+
+
+def validate_no_collisions_between_params_and_flags(flags):
+    if [flag for flag in flags if flag['name'] in FLAGS_LIST_TO_VALIDATE]:
+        raise CloudifyHelmSDKError(
+            'Please do not pass {flags_list} flags inside "flags" list'.format(
+                flags_list=FLAGS_LIST_TO_VALIDATE))
```

Comment:
Not clear to me what we should be passing. Maybe a more useful message?

Reply:
Done.